src/environments/environment.ts

```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  authUrl: 'https://landville-backend-web-api.herokuapp.com/api/v1/auth'
```

Comment:
Great Job.
Just one concern, the authUrl in your environment file should only be the global URL then it can be reused by every service that requires the URL.  Try adding this instead.``apiEndpoint: 'https://landville-backend-web-api.herokuapp.com/api/v1',``

Reply:
Thanks @bl4ck4ndbr0wn , I have worked on this